Alueta_MarkRoland/Activity3/Resume/index.css

```diff
@@ -0,0 +1,17 @@
+img {
```

Comment:
avoid global tag declaration for tags.
You can use class or id if you want to do specific layouts